### PR TITLE
[RM Ch04] Proofreading

### LINKED ISSUE #2715: [RM Ch04] Proofreading
Correct Typo's, grammar, etc.

Do not correct technical errors

### PR DESCRIPTION
**Pull request by pgoyal01**
Fixes #2715

### CODE REVIEW

File: doc/ref_model/chapters/chapter04.md [hunk 1]
```diff
@@ -34,53 +33,53 @@ a = Scope <(e)xternal | (i)nternal | (t)hird_party_instrumentation>
...
+**Exposed:** Refers to any object (e.g., resource discovery/configuration/consumption, platform telemetry, interface, etc.) that exists in or pertains to, the domain of the Cloud Infrastructure and is made visible (aka “Exposed”) to a workload. When an object is exposed to a given workload, the scope of visibility within a given workload is at the discretion of the workload’s designer. From an infrastructure perspective, the Infra-resident object is simply being exposed to one or more virtual environments (i.e., workloads). It is the responsibility of the kernel or supervisor/executive within the resource instance (VM or container) to control how, when and where the object is further exposed within the resource instance, with regard to permissions, security, etc. An object(s) is by definition visible within its origination domain.
```
Comment (ildikov) [inline doc/ref_model/chapters/chapter04.md:41]:
```suggestion
**Exposed:** Refers to any object (e.g., resource discovery/configuration/consumption, platform telemetry, interface, etc.) that exists in or pertains to, the domain of the Cloud Infrastructure and is made visible (aka “Exposed”) to a workload. When an object is exposed to a given workload, the scope of visibility within a given workload is at the discretion of the workload’s designer. From an infrastructure perspective, the Infra-resident object is simply being exposed to one or more virtual environments (i.e., workloads). It is the responsibility of the kernel or supervisor/executive within the resource instance (VM or container) to control how, when and where the object is further exposed within the resource instance, with regard to permissions, security, etc. An object(s) is by definition visible within its domain of origin.
```

File: doc/ref_model/chapters/chapter04.md [hunk 2]
```diff
@@ -202,10 +201,10 @@ This section covers a list of implicit Cloud Infrastructure capabilities and mea
 
 <p align="center"><b>Table 4-8:</b> Internal Measurement Capabilities of Cloud Infrastructure</p>
 
-<a name="4.1.5"></a>
+
 ### 4.1.5 Cloud Infrastructure Management Capabilities
 
-The Cloud Infrastructure Manager (CIM) is responsible for controlling and managing the Cloud Infrastructure compute, storage, and network resources. Resources allocation is dynamically set up upon workloads requirements. This section covers the list of capabilities offered by the CIM to workloads or service orchestrator.
+The Cloud Infrastructure Manager (CIM) is responsible for controlling and managing the Cloud Infrastructure compute, storage, and network resources. Resources are dybamically allocated based on workloads requirements. This section covers the list of capabilities offered by the CIM to workloads or service orchestrator.
```
Comment (ildikov) [inline doc/ref_model/chapters/chapter04.md:207]:
```suggestion
The Cloud Infrastructure Manager (CIM) is responsible for controlling and managing the Cloud Infrastructure compute, storage, and network resources. Resources are dybamically allocated based on workload requirements. This section covers the list of capabilities offered by the CIM to workloads or service orchestrator.
```

**Review by ildikov — COMMENTED** [review]
Just two small suggestions.

Thanks,
Ildikó

**Review by ildikov — APPROVED** [review]

**Review by walterkozlowski — APPROVED** [review]

**Comment by walterkozlowski** [comment]
@karinesevilla, @jbhartley - can you please review as well? Thanks.

File: doc/ref_model/chapters/chapter04.md [hunk 3]
```diff
@@ -34,53 +33,53 @@ a = Scope <(e)xternal | (i)nternal | (t)hird_party_instrumentation>
...
+This section describes a set of exposed Cloud Infrastructure capabilities and performance measurements. These capabilities and PMs are well known to workloads as they provide capabilities which workloads rely on.
 
 > _**Note**:  It is expected that Cloud Infrastructure capabilities and measurements will expand over time as more capabilities are added and technology enhances and matures._
 
-<a name="4.1.2.1"></a>
+
 #### 4.1.2.1 Exposed Resource Capabilities
 
-**Table 4-1** below shows resource capabilities of Cloud Infrastructure. Those indicate resources offered to workloads by Cloud Infrastructure.
+**Table 4-1** below shows resource capabilities of the Cloud Infrastructure avaiiable to workloads.
```
Comment (gkunz) [inline doc/ref_model/chapters/chapter04.md:62]:
```suggestion
**Table 4-1** below shows resource capabilities of the Cloud Infrastructure available to workloads.
```

File: doc/ref_model/chapters/chapter04.md [hunk 4]
```diff
@@ -202,10 +201,10 @@ This section covers a list of implicit Cloud Infrastructure capabilities and mea
 
 <p align="center"><b>Table 4-8:</b> Internal Measurement Capabilities of Cloud Infrastructure</p>
 
-<a name="4.1.5"></a>
+
 ### 4.1.5 Cloud Infrastructure Management Capabilities
 
-The Cloud Infrastructure Manager (CIM) is responsible for controlling and managing the Cloud Infrastructure compute, storage, and network resources. Resources allocation is dynamically set up upon workloads requirements. This section covers the list of capabilities offered by the CIM to workloads or service orchestrator.
+The Cloud Infrastructure Manager (CIM) is responsible for controlling and managing the Cloud Infrastructure compute, storage, and network resources. Resources are dybamically allocated based on workload requirements. This section covers the list of capabilities offered by the CIM to workloads or service orchestrator.
```
Comment (gkunz) [inline doc/ref_model/chapters/chapter04.md:207]:
```suggestion
The Cloud Infrastructure Manager (CIM) is responsible for controlling and managing the Cloud Infrastructure compute, storage, and network resources. Resources are dynamically allocated based on workload requirements. This section covers the list of capabilities offered by the CIM to workloads or service orchestrator.
```

**Review by gkunz — CHANGES_REQUESTED** [review]
two spelling corrections

**Comment by pgoyal01** [comment]
@gkunz Done - Thanks

**Comment by walterkozlowski** [comment]
@gkunz, fixed I can see.  Thanks!

**Comment by walterkozlowski** [comment]
@pgoyal01, can you please resolve the conflict?  Thanks.

**Comment by pgoyal01** [comment]
@walterkozlowski Resolved the conflicts. Thanks

**Comment by walterkozlowski** [comment]
@gkunz, can you please re-review and approve, if you can, thanks

**Review by gkunz — APPROVED** [review]

**Review by ulikleber — APPROVED** [review]

**Pull request merged**